### ISSUE TITLE
A11y: Active Inputs in Forms have insufficient color contrast

### ISSUE THREAD

**Issue by ThomasKindermann** [plone/volto#2570]
**Describe the bug**
Input text Color of active form input has insufficient color contrast of 3.29:1.
For better A11y it needs to have a color contrast of 4.5 or higher.

**To Reproduce**
1. Go to '/contact-form'
2. Click on any input eg: Name
3. Type anything
4. inspect input
5. check ":focus" in inspector
6. run axe dev tool
7. axe dev tool returns insufficient contrast violation

**Expected behavior**
contrast should be above 4.5:1

**Screenshots**
![Screenshot 2021-07-08 at 15 04 10](https://user-images.githubusercontent.com/50659809/124926398-c46b0500-dffd-11eb-9b22-9552d42b93d6.png)

**Comment by ThomasKindermann** [comment]
#1505

**Comment by ThomasKindermann** [comment]
@sneridagh @tisto 
To fix this issue we could make the Text black,

alternatively we could use #70787c instead of #878F93

![Screenshot 2021-07-08 at 16 01 34](https://user-images.githubusercontent.com/50659809/124935252-c8028a00-e005-11eb-9a53-23f5bf1deebd.png)

**Comment by ThomasKindermann** [comment]
PR for black text color: https://github.com/plone/volto/pull/2573

**Comment by davisagli** [comment]
Fixed in https://github.com/plone/volto/pull/6307
